integration/test/__main__.py

```diff
@@ -1,77 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
This will break the other nightlies.  They need to be reconfigured to use the unittest discover feature.